src/components/CentralContainer/Sidebar/SidebarTabs/PanelTitleButton.css

```diff
@@ -0,0 +1,5 @@
+.tab-panel-title-button {
+    z-index: var(--ylv-panel-title-button-z-index) !important;
```

Comment:
In fact, with `overflow-y: hidden;` added in `.sidebar-tabs`, do we still need this z-index override?